### PR TITLE
improvements for fix links script

### PR DESCRIPTION
**Pull request by azazellochg**

### CODE REVIEW

File: pyworkflow/project/project.py [hunk 1]
```diff
@@ -1957,37 +1957,31 @@ def setReadOnly(self, value):
...
-                                logger.info("    -> %s" % pwutils.red(sourceFile))
-
-                                newFile = pwutils.findFile(os.path.basename(sourceFile),
-                                                       searchDir,
-                                                       recursive=True)
-                                if newFile:
-                                    logger.info("  Found file %s, creating link... %s" % (newFile,
-                                        pwutils.green("   %s -> %s" % (f, newFile))))
-                                    pwutils.createAbsLink(newFile, f)
+            if prot.getClassName().startswith("ProtImport"):
```
Comment (pconesa) [inline pyworkflow/project/project.py:1965]: Not sure if all import protocols stars with ProtImport... but probably not all inherit from ProtImportBase

File: setup.py [hunk 2]
```diff
@@ -221,9 +221,6 @@
     # For example, the following would provide a command called `sample` which
     # executes the function `main` from this package when invoked:
     entry_points={
-        'console_scripts': [
-            'fix_links = pyworkflow.project.scripts.fix_links:main',
-        ],
         'pyworkflow.plugin': 'pyworkflowtests = pyworkflowtests'
```
Comment (pconesa) [inline setup.py:224]: Good point!

Comment (pconesa) [reply setup.py:224]: Maybe we can add other entry points?

File: pyworkflow/project/project.py (hunk 1 above)
Comment (azazellochg) [reply pyworkflow/project/project.py:1965]: ok, I will check all of them. I have not found any protocol inherited from ProtImportBase from pw. All are inherited from ProtImportImages from pwem, but we don't want to import pwem here..

File: setup.py (hunk 2 above)
Comment (azazellochg) [reply setup.py:224]: none of the project scripts have main() method. Of course, they all can be refactored to simplify the commands for users

File: pyworkflow/project/project.py (hunk 1 above)
Comment (azazellochg) [reply pyworkflow/project/project.py:1965]: All import protocols in pwem and tomo start with ProtImport

**Comment by azazellochg** [comment]
@pconesa anything else I need to do? :)

**Review by pconesa — APPROVED** [review]

**Pull request merged**